src/sqlfluff/dialects/dialect_teradata.py

```diff
@@ -450,9 +450,9 @@ class TdCreateTableOptions(BaseSegment):
     """
 
     type = "create_table_options_statement"
-    match_grammar = AnyNumberOf(
-        Sequence(
-            Ref("CommaSegment"),
+    match_grammar = Sequence(
+        Ref("CommaSegment"),
+        Delimited(
             OneOf(
```

Comment:
Is this OneOf() now unnecessary?

Reply:
Technically yes it is not needed as `Delimited` inherits from `OneOf`.

However, I always forget whether it inherits from `Sequence` or `OneOf` (I had to look it up again) so usually explicitly add the type I want underneath the `Delimited`. A bit of a waste, but more clear IMHO.

WDYT? Think I should just learn this properly and remove this unnecessary `OneOf`?

Reply:
Technically yes it is not needed as `Delimited` inherits from `OneOf`.

However, I always forget whether it inherits from `Sequence` or `OneOf` (I had to look it up again) so usually explicitly add the type I want underneath the `Delimited`. A bit of a waste, but more clear IMHO.

WDYT? Think I should just learn this properly and remove this unnecessary `OneOf`?

Reply:
Looks like I'm not the only one! There's 45 instances of `Delimited\(\n\s*OneOf` in our dialects.

As I say, I think it's clearer so would prefer to leave it. But can remove them all if you think it's not a good idea?